content/help/ops/traffic-management/deploy-guidelines/index.md

```diff
@@ -6,6 +6,35 @@ weight: 5
 
 This section provides specific deployment or configuration guidelines to avoid networking or traffic management issues.
 
+## 503 errors after setting destination rule
+
+If, after applying a `DestinationRule` for a service, requests to the service immediately start generating HTTP 503 errors
+and continue to do so until the `DestinationRule` is removed or reverted, it is possible that the `DesintationRule`
+has caused a TLS conflict for the service.
+
+If, for example, mutual TLS is globally configured in the cluster, the `DestinationRule` must include the following:
+
+{{< text yaml >}}
+  trafficPolicy:
+    tls:
+      mode: ISTIO_MUTUAL
+{{< /text >}}
+
+Otherwise, the mode will default to `DISABLED` which will cause client proxies (sidecars) to make plain HTTP,
+instead of TLS encrypted, requests. This will conflict with the server proxy, which is expecting encrypted requests.
+
+You can confirm that this has happened to a service if the `STATUS` field of the `istioctl authn tls-check` command
+is set to `CONFLICT`. For example:
+
+{{< text bash >}}
+$ istioctl authn tls-check httpbin.default.svc.cluster.local
+HOST:PORT                                  STATUS       SERVER     CLIENT     AUTHN POLICY     DESTINATION RULE
+httpbin.default.svc.cluster.local:8000     CONFLICT     mTLS       HTTP       default/         httpbin/default
+{{< /text >}}
+
+In this case, and whenever applying `DestinationRules`, make sure that the `trafficPolicy` TLS mode is set
```

Comment:
Do not modify the code reference to the plural, fore example, there is no `DestinationRules` only `DestinationRule`. Simplify and keep the code reference with:
Whenever you apply a `DestinationRule`, ensure the `trafficPolicy` TLS mode matches the global server configuration.